test/unit/imgix-provider.test.jsx

```diff
@@ -96,18 +96,38 @@ describe("ImgixProvider", () => {
       width: undefined,
     };
 
+    // expected props for background
+    const expectedBackgroundProps = {
+      domain: "sdk-test.imgix.net",
+      height: undefined,
+      imgixParams: undefined,
+      onMounted: undefined,
+      sizes: null,
+      src: "https://sdk-test.imgix.net/examples/pione.jpg",
+      width: undefined,
+    };
```

Comment:
suggestion(if-easy):
I think we can make this more maintainbale by using a shared props object for both tests.

suggestion:
```js
    const expectedProps = {
      disableSrcSet: false,
      domain: "sdk-test.imgix.net",
      height: undefined,
      imgixParams: undefined,
      onMounted: undefined,
      sizes: null,
      src: "https://sdk-test.imgix.net/examples/pione.jpg",
      width: undefined,
    };
    const expectedReactImgixProps = {
      ...expectedProps,
      disableSrcSet: false,
    };
    const expectedBgProps = {
      ...expectedProps,
    };
```

Reply:
That's a great call, good eye 🦅 👁️